app/assets/stylesheets/group.scss

```diff
@@ -0,0 +1,48 @@
+$white: #ffffff;
+
+.title {
+  color: $white;
+  font-size: 60px;
+  font-size: 5vw;
+  font-weight: bold;
+}
+.groups {
+  text-align: center;
+}
+
+.group {
+  margin: 20px;
```

Comment:
Properties should be ordered background-color, -moz-border-radius, -webkit-border-radius, border-radius, margin, max-width, padding